src/style.css

```diff
@@ -0,0 +1,25 @@
+* {
+  margin: 0;
+}
```

Comment:
Don't reset margin using * selector. Better specify the exact tags you want to reset